zcash_primitives/src/transaction/builder.rs

```diff
@@ -302,18 +306,20 @@ impl TransactionMetadata {
 }
 
 /// Generates a [`Transaction`] from its inputs and outputs.
-pub struct Builder<R: RngCore + CryptoRng> {
+pub struct Builder<P: consensus::Parameters, R: RngCore + CryptoRng> {
     rng: R,
+    height: u32,
     mtx: TransactionData,
     fee: Amount,
     anchor: Option<Fr>,
     spends: Vec<SpendDescriptionInfo>,
     outputs: Vec<SaplingOutput>,
     transparent_inputs: TransparentInputs,
     change_address: Option<(OutgoingViewingKey, PaymentAddress<Bls12>)>,
+    phantom: PhantomData<P>,
```

Comment:
i had to add this phantom field so the compiler wouldn't complain that `P: consensus::Parameters` was not being used.

Reply:
Once we switch to a parameters object in #261, this will likely just be replaced by that object.